use-timescale/integrations/terraform.md

```diff
@@ -1,17 +1,116 @@
 ---
-title: Terraform
-excerpt: Manage your Timescale services with Terraform
+title: Integrate Terraform with Timescale Cloud
+excerpt: Manage your Timescale Cloud services with a Terraform provider
 products: [cloud]
 keywords: [Terraform, configuration, deployment]
-tags: [terraform, manage]
+tags: [integrate]
 ---
 
-# Terraform
+import IntegrationPrereqs from "versionContent/_partials/_integration-prereqs.mdx";
 
-Terraform is an infrastructure-as-code tool that enables you to safely and predictably provision and manage infrastructure in any cloud. The Timescale Terraform provider allows you to manage Timescale services through a Terraform configuration.
+# Integrate Terraform with $CLOUD_LONG
 
-For more information about the supported service configurations and
-operations, see the
-[Timescale Terraform provider documentation][terraform-docs].
+[Terraform][terraform] is an infrastructure-as-code tool that enables you to safely and predictably provision and manage infrastructure. 
 
-[terraform-docs]: https://registry.terraform.io/providers/timescale/timescale/latest/docs
\ No newline at end of file
+This page explains how to configure Terraform to manage your $SERVICE_LONG or a self-hosted database. 
+
+## Prerequisites
+
+<IntegrationPrereqs />
+
+* [Download and install][terraform-install] Terraform.
+
+## Configure Terraform 
+
+Configure Terraform based on your deployment type:
+
+<Tabs label="Configure Terraform for your service">
+
+<Tab title="Timescale Cloud">
+
+You use the [$COMPANY Terraform provider][terraform-provider] to manage $SERVICE_LONGs:
+
+<Procedure>
+
+1. **Generate client credentials for programmatic use**
+
+   In [$CONSOLE][console], open `Project settings` and click `Create credentials`.
+
+1. **Configure $COMPANY Terraform provider**
+
+   Create a `main.tf` configuration file with the following content. Change `x.y.z` to the latest version of the provider. 
+
+   ```hcl
+   terraform {
+     required_providers {
+       timescale = {
+         source  = "timescale/timescale"
+         version = "x.y.z"
+       }
+     }
+   }
+
+   # Authenticate using client credentials generated in Timescale Console.
+   # When required, these credentials will change to a short-lived JWT to do the calls.
+   provider "timescale" {
+    project_id = var.ts_project_id
```

Comment:
Change this values to be somthing like

`project_id` = `<Your Timescale Project.Project ID>`

Reply:
I assume the 'var' part should stay, so 'var.<Your Timescale project ID>'?

Reply:
You declare variables here, you pass values in a separate file (the recommended way, there are a few). Added this as a step in the procedure.